tests/GridFS/BucketFunctionalTest.php

```diff
@@ -669,6 +670,19 @@ public function testUploadingAnEmptyFile(): void
         $this->assertSameDocument($expected, $fileDocument);
     }
 
+    public function testDisableMD5(): void
```

Comment:
While this test checks that the `disableMD5` option is used when passed to `uploadFromStream`, the PR title suggests the issue was with the option not being respected when passed to the constructor. Can you add a test for that as well?

Reply:
Indeed, test added.